<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>865</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(168, 168, 168);
background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:0, y2:1, stop:0 rgba(182, 185, 130, 255), stop:0.146893 rgba(252, 255, 187, 145), stop:0.333333 rgba(153, 104, 19, 208), stop:0.514124 rgba(158, 124, 37, 130), stop:0.683616 rgba(147, 125, 6, 130), stop:0.903409 rgba(200, 203, 152, 255));</string>
  </property>
  <widget class="graphs" name="graphs" native="true">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>1061</width>
     <height>551</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>640</y>
     <width>251</width>
     <height>211</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Terminal</family>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCheckBox" name="checkBox">
      <property name="font">
       <font>
        <family>Terminal</family>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: qradialgradient(spread:reflect, cx:1, cy:1, radius:2, fx:0.074, fy:0.051, stop:0 rgba(72, 72, 72, 252), stop:0.357955 rgba(1, 2, 3, 80), stop:0.44 rgba(12, 6, 6, 80), stop:0.5625 rgba(61, 56, 56, 156), stop:0.704545 rgba(72, 45, 45, 80), stop:0.943182 rgba(113, 113, 113, 0));</string>
      </property>
      <property name="text">
       <string>INSERTION SORT</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="font">
       <font>
        <family>Terminal</family>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Maximum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>10</width>
        <height>10</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_2">
      <property name="font">
       <font>
        <family>Terminal</family>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: qradialgradient(spread:reflect, cx:1, cy:1, radius:2, fx:0.074, fy:0.051, stop:0 rgba(72, 72, 72, 252), stop:0.357955 rgba(1, 2, 3, 80), stop:0.44 rgba(12, 6, 6, 80), stop:0.5625 rgba(61, 56, 56, 156), stop:0.704545 rgba(72, 45, 45, 80), stop:0.943182 rgba(113, 113, 113, 0));</string>
      </property>
      <property name="text">
       <string>BUBBLE SORT</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="font">
       <font>
        <family>Terminal</family>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Maximum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>10</width>
        <height>10</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_3">
      <property name="font">
       <font>
        <family>Terminal</family>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: qradialgradient(spread:reflect, cx:1, cy:1, radius:2, fx:0.074, fy:0.051, stop:0 rgba(72, 72, 72, 252), stop:0.357955 rgba(1, 2, 3, 80), stop:0.44 rgba(12, 6, 6, 80), stop:0.5625 rgba(61, 56, 56, 156), stop:0.704545 rgba(72, 45, 45, 80), stop:0.943182 rgba(113, 113, 113, 0));</string>
      </property>
      <property name="text">
       <string>MERGE SORT</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_3">
      <property name="font">
       <font>
        <family>Terminal</family>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Maximum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>10</width>
        <height>10</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_4">
      <property name="font">
       <font>
        <family>Terminal</family>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: qradialgradient(spread:reflect, cx:1, cy:1, radius:2, fx:0.074, fy:0.051, stop:0 rgba(72, 72, 72, 252), stop:0.357955 rgba(1, 2, 3, 80), stop:0.44 rgba(12, 6, 6, 80), stop:0.5625 rgba(61, 56, 56, 156), stop:0.704545 rgba(72, 45, 45, 80), stop:0.943182 rgba(113, 113, 113, 0));</string>
      </property>
      <property name="text">
       <string>RADIX SORT</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>640</y>
     <width>251</width>
     <height>211</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Terminal</family>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QCheckBox" name="checkBox_5">
      <property name="font">
       <font>
        <family>Terminal</family>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: qradialgradient(spread:reflect, cx:1, cy:1, radius:2, fx:0.074, fy:0.051, stop:0 rgba(72, 72, 72, 252), stop:0.357955 rgba(1, 2, 3, 80), stop:0.44 rgba(12, 6, 6, 80), stop:0.5625 rgba(61, 56, 56, 156), stop:0.704545 rgba(72, 45, 45, 80), stop:0.943182 rgba(113, 113, 113, 0));</string>
      </property>
      <property name="text">
       <string>BUCKET SORT</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_4">
      <property name="font">
       <font>
        <family>Terminal</family>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Maximum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>10</width>
        <height>10</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_6">
      <property name="font">
       <font>
        <family>Terminal</family>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: qradialgradient(spread:reflect, cx:1, cy:1, radius:2, fx:0.074, fy:0.051, stop:0 rgba(72, 72, 72, 252), stop:0.357955 rgba(1, 2, 3, 80), stop:0.44 rgba(12, 6, 6, 80), stop:0.5625 rgba(61, 56, 56, 156), stop:0.704545 rgba(72, 45, 45, 80), stop:0.943182 rgba(113, 113, 113, 0));</string>
      </property>
      <property name="text">
       <string>COUNTING SORT</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_5">
      <property name="font">
       <font>
        <family>Terminal</family>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Maximum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>10</width>
        <height>10</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_7">
      <property name="font">
       <font>
        <family>Terminal</family>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: qradialgradient(spread:reflect, cx:1, cy:1, radius:2, fx:0.074, fy:0.051, stop:0 rgba(72, 72, 72, 252), stop:0.357955 rgba(1, 2, 3, 80), stop:0.44 rgba(12, 6, 6, 80), stop:0.5625 rgba(61, 56, 56, 156), stop:0.704545 rgba(72, 45, 45, 80), stop:0.943182 rgba(113, 113, 113, 0));</string>
      </property>
      <property name="text">
       <string>QUICK SORT</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_6">
      <property name="font">
       <font>
        <family>Terminal</family>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Maximum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>10</width>
        <height>10</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_8">
      <property name="font">
       <font>
        <family>Terminal</family>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: qradialgradient(spread:reflect, cx:1, cy:1, radius:2, fx:0.074, fy:0.051, stop:0 rgba(72, 72, 72, 252), stop:0.357955 rgba(1, 2, 3, 80), stop:0.44 rgba(12, 6, 6, 80), stop:0.5625 rgba(61, 56, 56, 156), stop:0.704545 rgba(72, 45, 45, 80), stop:0.943182 rgba(113, 113, 113, 0));</string>
      </property>
      <property name="text">
       <string>HEAP SORT</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_7">
      <property name="font">
       <font>
        <family>Terminal</family>
       </font>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Maximum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>10</width>
        <height>10</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_9">
      <property name="font">
       <font>
        <family>Terminal</family>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: qradialgradient(spread:reflect, cx:1, cy:1, radius:2, fx:0.074, fy:0.051, stop:0 rgba(72, 72, 72, 252), stop:0.357955 rgba(1, 2, 3, 80), stop:0.44 rgba(12, 6, 6, 80), stop:0.5625 rgba(61, 56, 56, 156), stop:0.704545 rgba(72, 45, 45, 80), stop:0.943182 rgba(113, 113, 113, 0));</string>
      </property>
      <property name="text">
       <string>SELECTION SORT</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>640</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Terminal</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qradialgradient(spread:reflect, cx:1, cy:1, radius:2, fx:0.074, fy:0.051, stop:0 rgba(72, 72, 72, 252), stop:0.357955 rgba(1, 2, 3, 80), stop:0.44 rgba(12, 6, 6, 80), stop:0.5625 rgba(61, 56, 56, 156), stop:0.704545 rgba(72, 45, 45, 80), stop:0.943182 rgba(113, 113, 113, 0));
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>COMPARE CHOSEN METHODS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>640</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Terminal</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qradialgradient(spread:reflect, cx:1, cy:1, radius:2, fx:0.074, fy:0.051, stop:0 rgba(72, 72, 72, 252), stop:0.357955 rgba(1, 2, 3, 80), stop:0.44 rgba(12, 6, 6, 80), stop:0.5625 rgba(61, 56, 56, 156), stop:0.704545 rgba(72, 45, 45, 80), stop:0.943182 rgba(113, 113, 113, 0));
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>COMPARE ALL METHODS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>780</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/img_185728.png</normaloff>:/icons/icons/img_185728.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>125</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>0</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/25694 (1).png</normaloff>:/icons/icons/25694 (1).png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>125</height>
    </size>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>graphs</class>
   <extends>QWidget</extends>
   <header location="global">graphs.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
